feature/home/src/main/java/org/sopt/at/home/HomeScreen.kt

```diff
@@ -91,19 +116,44 @@ fun HomeScreen(
             }
         }
         stickyHeader {
```

Comment:
스티키헤더 사용해서 탭을 이런식으로 구현하는거군요 !! 배우고 갑니다.. 🏃

Reply:
그런데 lazycolumn에 stickyHeader는 아직 실험중인 api라서
```suggestion
       @OptIn(ExperimentalFoundationApi::class)
```
상위 컴포저블에 어노테이션을 달아줘야 합니다. 릴리즈 목적의 앱이면 @OptIn을 피하는게 좋다고 생각하는데 과제라서 써봤습니다ㅎㅎㅎㅎㅎ